src/entities/endorsement.cpp

```diff
@@ -71,6 +78,11 @@ void VbkEndorsement::toVbkEncoding(WriteStream& stream) const {
     stream.writeBE<uint8_t>(containingHash[i]);
   }
   stream.write(blockOfProof);
+
+  stream.writeBE<uint32_t>((uint32_t)payoutInfo.size());
```

Comment:
maybe use `writeVarLenValue`? it will be same, but easier to read

Reply:
It won't be the same. Additional byte with data length size will be appended this way.

Reply:
but for varlen it is also the case?
https://github.com/VeriBlock/alt-integration-cpp/blob/master/src/serde.cpp#L52

if u want to have single byte len, use 
https://github.com/VeriBlock/alt-integration-cpp/blob/master/src/serde.cpp#L47

Reply:
We store length = 4 bytes + data. writeVarLenValue stores size = 1 byte + length = 4bytes + data. Changing to writeVarLenValue will break compatibility but we may start using it.

Reply:
@blackyblack there's no existing databases to break compatibility with, so you can safely change this serialization code